### PR TITLE
updated oqtane release notes, added .NEXT and fido2-net-lib as newly approved Foundation Member Projects

### PR DESCRIPTION
**Pull request by sbwalker**

### CODE REVIEW

File: input/blog/posts/_current-newsletter-draft.md
Comment (BethMassi) [inline input/blog/posts/_current-newsletter-draft.md:20]: Can you or @ChrisSfanos get the logos for these and put them in the table format like the other new projects?

**Comment by ChrisSfanos** [comment]
So technically I've avoided putting projects in the newsletter until after the CLA has been signed.  So if we want we can hold these off until the next one.  I'll leave it up to @sbwalker or @BethMassi to make a business decision on it

**Comment by BethMassi** [comment]
Sounds good to me @ChrisSfanos. @sbwalker can you submit the PR with just the update you want for Oqtane?

**Review by Arhell — APPROVED** [review]

**Comment by sbwalker** [comment]
It appears that Troy chose the BSD3 permissive license - this PR should be adjusted

**Comment by clairernovotny** [comment]
There's a conflict here too

**Comment by clairernovotny** [comment]
looks like this should be rebased once I merge the blog in a few minutes and the hibp md and jpg can be elided

**Pull request closed without merging**
